lib/vsc/utils/nagios.py

```diff
@@ -408,6 +409,43 @@ def __str__(self):
         return "%s | %s" % (self.message, ' '.join(perf))
 
 
+class FakeSimpleNagios(NagiosResult):
+    """Absolutely no side effects.
+
+    For use in long running services that have no access to the locations where
```

Comment:
such as?

Reply:
xdmod-consumer

Reply:
and why wouldn;t it have access to the nagios locations?

Reply:
because it is running as the XDMoD user

Reply:
hmmmm
we had stuff in the past like this. i think we settled on "running as root is better than running without monitoring".
i guess we can make it write into paths it owns, we can always see how to have nrpe actually read it out